saos-webapp/src/main/webapp/WEB-INF/view/judgment/judgmentList.jsp

```diff
@@ -63,7 +63,16 @@
 				</span>
 				<ul>
 					<c:forEach items="${judgment.judges}" var="judge" >
-						<li class="judge"><c:out value="${judge.name}" /></li>
+				
+						<li >
+							<span class="judge"><c:out value="${judge.name}" /></span>
+							
+							<c:if test="${judge.presidingJudge}" >
```

Comment:
I think in *.jsp you can only use methods(getters) that starts with "get" or "is".